defaults/main.yml

```diff
@@ -68,23 +69,25 @@ camac_base_domain: "camac.ch"
 # camac base directories #
 ##########################
 
+camac_python_basedir: "/usr/share/camac"
```

Comment:
is it a good idea to have the same default for python and php base dir?

Reply:
I will check if this variable is actually required.

Reply:
This should be fixed now. I've changed the basedir path of the python service to `/usr/share/camac-api/`.